src/features/mint/components/MultipleDepositsHelpers.tsx

```diff
@@ -283,16 +438,76 @@ export const DepositNavigation: FunctionComponent<DepositNavigationProps> = ({
           const requiresAction =
             depositStatus === DepositEntryStatus.ACTION_REQUIRED;
           const completed = depositStatus === DepositEntryStatus.COMPLETED;
-          const confirmationProps = completed
-            ? {}
-            : {
-                confirmations: lockConfirmations,
-                targetConfirmations: lockTargetConfirmations,
-              };
+          const completing = depositStatus === DepositEntryStatus.COMPLETING;
+          const confirmationProps =
+            completed || completing
+              ? {}
+              : {
+                  confirmations: lockConfirmations,
+                  targetConfirmations: lockTargetConfirmations,
+                };
+
+          let InfoContent: any = null;
+          if (depositStatus === DepositEntryStatus.COMPLETED) {
+            InfoContent = (
+              <div>
+                <Typography variant="body1" color="textPrimary">
+                  {lockTxAmount} {lockCurrencyConfig.short}
+                </Typography>
+                <Typography variant="body2" color="primary">
+                  {t("mint.deposit-navigation-completed-label")}
+                </Typography>
+              </div>
+            );
+          } else if (depositStatus === DepositEntryStatus.COMPLETING) {
+            InfoContent = (
+              <div>
+                <Typography variant="body1" color="textPrimary">
+                  {lockTxAmount} {lockCurrencyConfig.short}
+                </Typography>
+                <Typography variant="body2" color="textSecondary">
+                  {t("mint.deposit-navigation-completing-label")}
+                </Typography>
+              </div>
+            );
+          } else if (depositStatus === DepositEntryStatus.PENDING) {
+            InfoContent = (
+              <div>
+                <Typography variant="body1" color="textPrimary">
+                  {lockTxAmount} {lockCurrencyConfig.short}
+                </Typography>
+                {lockTargetConfirmations !== 0 ? (
+                  <Typography variant="body2" color="textSecondary">
+                    {t("mint.deposit-navigation-confirmations-label", {
+                      confirmations: lockConfirmations,
+                      targetConfirmations: lockTargetConfirmations,
+                    })}
+                  </Typography>
+                ) : (
+                  <Skeleton variant="text" width={100} height={14} />
+                )}
+              </div>
+            );
+          } else if (depositStatus === DepositEntryStatus.ACTION_REQUIRED) {
+            InfoContent = (
+              <div>
+                <Typography variant="body1" color="textPrimary">
+                  {lockTxAmount} {lockCurrencyConfig.short}
+                </Typography>
+                <Typography variant="body2" color="primary">
+                  {t("mint.deposit-navigation-ready-to-mint-label")}
```

Comment:
This appears to be shown also while the tx is being submitted to RenVM, so it would be better to not having anything at all if we can't differentiate the two states in this component.